docs/streams/developer-guide/dsl-api.html

```diff
@@ -640,11 +640,39 @@ <h4><a id="streams_concepts_globalktable" href="#streams_concepts_globalktable">
       <span class="n">Serdes</span><span class="o">.</span><span class="na">String</span><span class="o">(),</span> <span class="cm">/* key (note: type was modified) */</span>
       <span class="n">Serdes</span><span class="o">.</span><span class="na">Integer</span><span class="o">())</span> <span class="cm">/* value (note: type was modified) */</span>
   <span class="o">);</span>
+                            </pre></div>
+                            </div>
+                        </td>
+                    </tr>
+                    <tr class="row-even"><td><p class="first"><strong>Cogroup</strong></p>
+                        <ul class="last simple">
+                            <li>KGroupedStream &rarr; CogroupedKStream</li>
+                            <li>CogroupedKStream &rarr; CogroupedKStream</li>
+                        </ul>
+                    </td>
+                        <td><p class="first">Cogrouping allows to aggregate multiple input streams in a single operation.
+                            The different (already grouped) input streams must have the same key type and may have different values types.
+                            <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/kstream/KGroupedStream.html#cogroup">KGroupedStream#cogroup()</a> creates a new cogrouped stream with a single input stream, while <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/kstream/CogroupedKStream.html#cogroup">CogroupedKStream#cogroup()</a> adds a grouped stream to an existing cogrouped stream.
+                            A <code>CogroupedKStream</code> may be <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/kstream/CogroupedKStream.html#windowedBy">windowed</a> before it is <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/kstream/CogroupedKStream.html#aggregate"> aggregated</a> (<a class=" reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/kstream/CogroupedKStream.html#aggregate--">aggregate details</a>) and (<a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/kstream/CogroupedKStream.html#windowedBy--">windowBy details</a>)<p>
```

Comment:
If you link from `windowed` and `aggregated` to the JavaDocs in the "main" sentence, there is no need to link to the same methods via `(Detailed...)`, ie:
```
(<a class=" reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/kstream/CogroupedKStream.html#aggregate--">aggregate details</a>) and (<a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/kstream/CogroupedKStream.html#windowedBy--">windowBy details</a>)
```

I don't care what you chose, but avoid double linking :)

Nit, there is a whitespace in the second link: `> aggregated</a>` that should not be there.

The tag at the end should be closing tag? `</p>` instead of `<p>` ?